extensions/shared/external-media/sources/with-media.js

```diff
@@ -207,7 +207,7 @@ export default function withMedia() {
 
 				return (
 					<Modal
-						onRequestClose={ ev => onClose( ev ) }
+						onRequestClose={ event => onClose( event ) }
```

Comment:
why not just ...

```suggestion
						onRequestClose={ onClose }
```
?